main.js

```diff
@@ -155,3 +170,16 @@ if (portfolioItems) {
   });
 }
 
+// start of form validation
+form.addEventListener('keyup', (e) => {
+  e.preventDefault();
+  // validate the form fields
+  const email = document.querySelector('#email').value;
+  // check if name field is empty
+  if (email !== email.toLowerCase()) {
+    const errorDiv = document.querySelector('#validate');
+    errorDiv.innerText = 'Kindly type email address in lowercase';
+  } else {
+    form.submit();
```

Comment:
- [ ] Your form authomatically submits by only typing the email because of the implementation you have done here. The form is only to be submitted when the user clicks submit after filling correctly the fields provided> I kindly suggest you make the appropriate corrections to your functionality 👍 
- [ ] Rather than using  a `keyup` event in `line 174`, make use of a `submit` eventlistener and your application would act required. 👍 ✨